upup/models/cloudup/resources/addons/networking.projectcalico.org/k8s-1.16.yaml.template

```diff
@@ -61,232 +65,3131 @@ data:
 ---
 # Source: calico/templates/kdd-crds.yaml
 
-apiVersion: apiextensions.k8s.io/v1beta1
+
+---
+apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: (devel)
```

Comment:
Seems suspicious to have a version of "(devel)"

Reply:
Yeah, seems to be a small mess in the official templates.

Reply:
Calico is moving to install via operator but still provides the manifests. Probably this is why the mess. At some point we should look into that and see if we can convert our templates also.